src/components/BookList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { removeItem } from '../redux/books/bookSlice'
+import { useDispatch } from 'react-redux';
+// import {store} from '../redux/store'
```

Comment:
This change is optional because it wasn't mentioned by the previous reviewer.
[OPTIONAL]
- [ ] I kindly suggest you remove commented-out code from your codebase as it doesn't make it look neat.